nautobot/extras/api/fields.py

```diff
@@ -18,11 +15,3 @@ def __init__(self, **kwargs):
     def to_internal_value(self, data):
         set_value = super().to_internal_value(data)
         return sorted(set_value)
-
-
-class RoleSerializerField(LimitQuerysetChoicesSerializerMixin, NestedRoleSerializer):
-    """NestedSerializer field for `Role` object fields."""
-
-
-class StatusSerializerField(LimitQuerysetChoicesSerializerMixin, NestedStatusSerializer):
-    """NestedSerializer field for `Status` object fields."""
```

Comment:
Don't we still need these for the `LimitQuerysetChoicesSerializerMixin` functionality?

Reply:
Im not too sure if we still need `LimitQuerysetChoicesSerializerMixin` or what its original functionality is. (for example is it trying to limit the instances of `Statuses` those that are valid for `Location` for a `POST` request?) If so, it seems like `WritableSerializerMixin` is already doing that.

Reply:
Interesting. The mixin was added in #2822, I assume there was some need for it. Maybe since we have `ForeignKeyLimitedByContentTypes` we don't need it any more? @timizuoebideri1 any thoughts?